AUDIO_GET_PTS is removed with linux-libc-headers 4.19.x
Define AUDIO_GET_PTS, based on:
https://elixir.bootlin.com/linux/v4.9.12/source/include/uapi/linux/dvb/audio.h#L132

diff --git a/output/linuxdvb_mipsel.c b/output/linuxdvb_mipsel.c
index 00b5992..a7a5b79 100755
--- a/output/linuxdvb_mipsel.c
+++ b/output/linuxdvb_mipsel.c
@@ -54,6 +54,7 @@
 
 #define cERR_LINUXDVB_NO_ERROR      0
 #define cERR_LINUXDVB_ERROR        -1
+#define AUDIO_GET_PTS              _IOR('o', 19, __u64)
 
 static const char VIDEODEV[] = "/dev/dvb/adapter0/video0";
 static const char AUDIODEV[] = "/dev/dvb/adapter0/audio0";
